nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -266,6 +347,22 @@ def migrate_site_and_region_data_to_locations(apps, schema_editor):
             note.assigned_object_id = region_loc.id
         Note.objects.bulk_update(notes, ["assigned_object_type", "assigned_object_id"], 1000)
 
+        object_changes_changed = ObjectChange.objects.filter(changed_object_type=region_ct)
+        for oc in object_changes_changed:
+            oc.changed_object_type = location_ct
+            region = Region.objects.get(id=oc.changed_object_id)
+            region_loc = Location.objects.get(name=region.name, location_type=region_lt)
```

Comment:
Here too, looks like we now have three consecutive cases where we're looking up Regions and their equivalent Locations on the fly for every iteration. Could we instead gather this information up in advance so that we don't have to look it up over again for each individual ObjectChange record?